.travis.yml

```diff
@@ -23,7 +23,8 @@ branches:
 addons:
   apt:
     sources:
-    - sourceline: 'deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic main'
+# TODO: change links to bionic when packages there will not crash travis
+    - sourceline: 'deb http://apt.llvm.org/xenial/ llvm-toolchain-xenial-10 main'
```

Comment:
The following line should work fine:
`- sourceline: 'deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-10 main'`